### ISSUE TITLE
using get_accounts method, I cannot see USDC or EUR wallets

### ISSUE THREAD

**Issue by samdetrent** [coinbase/coinbase-advanced-py#54]
I'm using this API mostly with success. One pesky issue is, some of my accounts seem to be unavailable through the API when using get_accounts, but they are available when using the Coinbase mobile app or website? Specifically USDC and EUR wallets I have seen in the past using the get_accounts method, but today, neither are visible.

Why is that? Thanks :)

**Comment by ilikecheese** [comment]
Yep, me too, I can't see BTC, but can see all others. I'll check the discord for help.
